### DISCUSSION TITLE
Combobox - add quick way to clear selection

### DISCUSSION THREAD

**Discussion by dexfab** [vmware/clarity#5641] [Ideas]
Add an x once user has made a selection so it can cleared quickly. This is especially valid when using the single selection option as currently the only way to change the selection is to delete the text in the input first and this can be quite frustrating.

ng-select has a good example of this feature https://ng-select.github.io/ng-select#/multiselect

**Comment by Jinnie** [comment]
Thanks for the suggestion, @dexfab 
We are very careful about adding new features to the Combobox, as it already has too complicated accessibility.
But we will have your suggestion under consideration for the future Clarity Core implementation of the Combobox.

**Reply by d-m-s** [reply]
We had the same feedback from our users. They found a reselection very un-intuitive. 
Described in more detail in #6469 

So I also believe this feature could take away a lot of pain for the users when re-selecting.
